packages/file-types/zarr/src/anndata-loaders/ObsFeatureMatrixAnndataLoader.js

```diff
@@ -10,6 +10,19 @@ import {
 // value returned from the load function.
 const toObject = data => ({ data });
 
+const filterOutEveryOther = (arr) => {
+  const filtered = new arr.constructor(arr.length / 2);
+  for (let i = 0; i < (arr.length / 2); i++) {
+    filtered[i] = arr[i * 2];
+  }
+  return filtered;
+};
+
+// eslint-disable-next-line no-undef
+const maybeDowncastInt64 = data => (data.constructor === BigInt64Array
```

Comment:
Could we add a test for this? Something where we pass in a BigInt64 array with values in the Int32 range as input and we get back an array with the same values